kotlin/src/main/com/looker/rtl/ErrorDoc.kt

```diff
@@ -116,7 +116,7 @@ class ErrorDoc(val sdk: APIMethods, val cdnUrl: String = ErrorCodesUrl): IErrorD
 
         /** API error document_url link pattern */
         private const val ErrorDocPatternExpression =
-            """(?<redirector>https:\/\/docs\.looker\.com\/r\/err\/)(?<apiVersion>.*)\/(?<statusCode>\d{3})(?<apiPath>.*)"""
+            """(?<redirector>https://docs\.looker\.com/r/err/|https://cloud\.google\.com/looker/docs/r/err/)(?<apiVersion>.*)/(?<statusCode>\d{3})(?<apiPath>.*)"""
```

Comment:
Do we no longer need to escape here `\/`?

Reply:
tests work, IDEA complained about the escaping. Maybe it's a Kotlin thing.